pkg/registry/apis/dashboard/register.go

```diff
@@ -628,6 +633,28 @@ func (b *DashboardsAPIBuilder) storageForVersion(
 	return nil
 }
 
+func (b *DashboardsAPIBuilder) afterDelete(obj runtime.Object, _ *metav1.DeleteOptions) {
+	if b.resourcePermissionsSvc == nil {
```

Comment:
nit: I remember we had a function called `isInterfaceNil` somewhere, which made us able to not use a pointer to an interface - do you think we should use it?

Reply:
good idea - definitely. updated!